tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -2,7 +2,6 @@ tests:
   - test:
       tags: kernel
   - test_tickless:
-      build_only: true
       extra_args: CONF_FILE="prj_tickless.conf"
-      filter: CONFIG_BOARD_QEMU_X86
+      filter: not CONFIG_PULPINO_TIMER and not CONFIG_ALTERA_AVALON_TIMER and not CONFIG_RISCV_MACHINE_TIMER
```

Comment:
why not just exclude arches riscv32 and nios2 instead of this long filter? It will end up the same but faster in execution.

Reply:
sure, I agree. will update accordingly.